docs/content/docs/install/settings.md

```diff
@@ -72,29 +72,39 @@ There is a few things you can configure through the config map
   for your repository and create a Repository CR.
 
   This feature is disabled by default and is only supported with GitHub App.
-  
+
   ****NOTE****: If you have a GitHub App already setup then verify if `Repository` event is subscribed.
 
 * `auto-configure-repo-namespace-template`
 
   If `auto-configure-new-github-repo` is enabled then you can provide a template for generating the namespace for your new repository.
   By default, the namespace will be generated using this format `{{repo_name}}-pipelines`.
-  
+
   You can override the default using the following variables
-  
+
   * `{{repo_owner}}`: The repository owner.
   * `{{repo_name}}`: The repository name.
-  
+
   for example. if the template is defined as `{{repo_owner}}-{{repo_name}}-ci`, then the namespace generated for repository
   `https://github.com/owner/repo` will be `owner-repo-ci`
 
+* `error-log-snippet`
+
```

Comment:
Should we add a point to say that its enabled by default?

Reply:
i think it's implicit unless disabled? that's how other things in documentation works?